packages/office-ui-fabric-react/src/components/SwatchColorPicker/examples/SwatchColorPicker.Basic.Example.tsx

```diff
@@ -45,6 +45,19 @@ export class SwatchColorPickerBasicExample extends React.Component<any, IBasicSw
             ]
           }
         />
+        <div>Simple square swatch color picker with theme colors:</div>
+        <SwatchColorPicker
+          columnCount={ 4 }
+          cellShape={ 'square' }
+          colorCells={
+            [
+              { id: 'a', label: 'green', color: SwatchThemeColor.ThemePrimary },

```

Comment:
Why is the label green? Are you sure that's the primary? Why not use `primary`

Reply:
left by mistake